binding/Binding/SKGlyphWarping.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace SkiaSharp
+{
+	/// <summary>
+	/// How the geometry of glyphs should be warped when drawn on a path.
+	/// </summary>
+	public enum SKGlyphWarping
```

Comment:
Indeed, this can just be a boolean, I will simplify. I have the habit of always using enums because it make the calling code more readable unless named parameters are used.

Reply:
I added `wrapGlyphs = true`, but after the `paint` argument, otherwise a default value isn't possible, and I would have to add way to many overloads. Since the warping influences how the glyphs are painted, it felt correct to put it after the paint. Can be refactored if you want.